src/main/java/com/contrastsecurity/sdk/scan/CodeArtifactImpl.java

```diff
@@ -52,6 +53,12 @@ public String filename() {
     return inner.filename();
   }
 
+  @Override
+  @Nullable
```

Comment:
Do you need `@Nullable` here? I believe it should be inherited from the interface definition.

Reply:
I removed this and everything still works... so I guess not.